docs/docs/migration-guide.mdx

```diff
@@ -14,6 +14,17 @@ how you can migrate from one version to another.
 
 ### Machine Learning Components
 
+### `DIET`, `TED`, and `ResponseSelector`
```

Comment:
Is this under `### Machine Learning Components`? if so should be `####` I guess?

Reply:
🤦 Right.